plugins/select/README.md

```diff
@@ -200,6 +198,24 @@ connect(select(models => {
 }))(...)
 ```
 
+Here is a full example combining raw, uncomputed state with selected state:
```

Comment:
Maybe lead into this with motivation:

"You might want to combine raw, uncomputed state with your selectors. In this case, the combined selector from select can be called directly"

I'll have to fix some of the language here around select itself later, so maybe just leave it for that.